be/src/storage/lake/update_manager.cpp

```diff
@@ -34,6 +34,7 @@ namespace starrocks::lake {
 UpdateManager::UpdateManager(LocationProvider* location_provider, MemTracker* mem_tracker)
         : _index_cache(std::numeric_limits<size_t>::max()),
           _update_state_cache(std::numeric_limits<size_t>::max()),
+          _compaction_cache(std::numeric_limits<size_t>::max()),
```

Comment:
Is there an upper limit on cache memory usage?

Reply:
Yes, it is controls by `update` mem tracker